doc/content/integrations/payload-formatters/cayenne.md

```diff
@@ -36,3 +36,19 @@ will produce the following `uplink_message`:
   }
 }
 ```
+
+A description of the Cayenne LPP sensor types is available in the Cayenne documentation [here](https://developers.mydevices.com/cayenne/docs/lora/#lora-cayenne-low-power-payload-reference-implementation-cayenne-lpp-cc-constants-definitions). {{% tts %}} uses the sensor names to create the `decoded_payload` fields, so a Cayenne payload with one `TEMPERATURE` and one `BAROMETRIC_PRESSURE` sensor reading will produce the following decoded payload:
```

Comment:
Also, it would be great to briefly describe how cayenne lpp sensor names translate to `decoded_payload` entries similar to [oidc example](https://thethingsstack.io/reference/federated-auth/oidc/#how-are-usernames-generated-for-external-users-) in the docs.

Reply:
For reference, these are the support payload types (by key name):
https://github.com/TheThingsNetwork/lorawan-stack/blob/9f9745e166c9afe168a7678fa2720f283c449f15/pkg/messageprocessors/cayennelpp/keys.go#L26-L38

The format is `{key}_{channel}`.